src/commands/publish/package.rs

```diff
@@ -38,7 +38,7 @@ impl Package {
 
         let package_json: String = fs::read_to_string(manifest_path.clone())?.parse()?;
         let package: Package = serde_json::from_str(&package_json)
-            .expect(&format!("could not parse {:?}", manifest_path));
+            .unwrap_or_else(|_| panic!("could not parse {:?}", manifest_path));
```

Comment:
Out of curiosity, why did `expect` cause these warnings?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#expect_fun_call